tests/projects/workspace/workspace.test.js

```diff
@@ -1,10 +1,10 @@
 import { fileURLToPath } from 'node:url'
 import { test } from 'uvu'
 import * as assert from 'uvu/assert'
-import {crawlFrameworkPkgs} from '../../../src/index.js'
+import { crawlFrameworkPkgs } from '../../../src/index.js'
 
-const workspaceRoot = fileURLToPath(new URL('./', import.meta.url)).replace(/\/$/,'')
-const root = `${workspaceRoot}/packages/workspace-app`;
+const workspaceRoot = fileURLToPath(new URL('../../../', import.meta.url)).replace(/\/$/,'')
```

Comment:
the tests workspaceRoot is `tests/projects/workspace` we might have to add a file to that dir so that yarn detects it as root?